normal_estimator:
    ros__parameters:
        MAX_RANGE: 120.0
